### PR TITLE
Move notebooks from aperturedb-python

### PR DESCRIPTION
**Pull request by bovlb**
With this change I have moved the notebooks from aperturedb-python, obviating https://github.com/aperture-data/aperturedb-python/pull/512
<s>Website crawl notebook to follow shortly.</s>
Website crawl notebooks added.

### CODE REVIEW

File: notebooks/semantic_search/cohere_wikipedia_ingest.ipynb [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+    "See our [Configuration](https://docs.aperturedata.io/Setup/client/configuration) help page for more options."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! adb config create rag --from-json --active "
```
Comment (vishakha041) [inline notebooks/semantic_search/cohere_wikipedia_ingest.ipynb:148]: This needs to change now to remove name?

File: notebooks/semantic_search/cohere_wikipedia_search.ipynb [hunk 2]
```diff
@@ -0,0 +1,516 @@
...
+    "    display(Markdown(\"\\n\".join([\n",
+    "        f\"### RAG Answer\\n{rag_answer['answer']}\",\n",
+    "        f\"### Documents\",\n",
+    "        *(f\"{i}. **[{doc.metadata['title']}]({doc.metadata['url']})**: {doc.page_content}\" for i, doc in enumerate(rag_answer[\"context\"], 1) )\n",
+    "        ])))\n",
+    "\n",
+    "\n",
+    "interactive = True\n",
+    "if interactive:\n",
+    "    input = Combobox(\n",
```
Comment (vishakha041) [inline notebooks/semantic_search/cohere_wikipedia_search.ipynb]: There was a comment you asked a question in the python repo - what I meant was, showing the response to the question from the dropdown -- RAG / Non RAg like it shows when we execute in the notebook.

File: notebooks/semantic_search/cohere_wikipedia_ingest.ipynb (hunk 1 above)
Comment (bovlb) [reply notebooks/semantic_search/cohere_wikipedia_ingest.ipynb:148]: We can't do that until https://github.com/aperture-data/aperturedb-python/pull/510 is released.

File: notebooks/semantic_search/cohere_wikipedia_search.ipynb (hunk 2 above)
Comment (bovlb) [reply notebooks/semantic_search/cohere_wikipedia_search.ipynb]: OK.  I'll take a look.

The `interactive` stuff is for CI test runs of notebooks, which we don't yet have on the new repo.

Comment (bovlb) [reply notebooks/semantic_search/cohere_wikipedia_search.ipynb]: I've refreshed the outputs of the two WIkipedia notebooks.  Unfortunately, I think the way the last cell uses widgets means that the output doesn't get saved.  Maybe I can de-widget it.

File: notebooks/semantic_search/cohere_wikipedia_ingest.ipynb (hunk 1 above)
Comment (vishakha041) [reply notebooks/semantic_search/cohere_wikipedia_ingest.ipynb:148]: @gsaluja9 - could you please resolve this dependency?

File: notebooks/semantic_search/website_ingest.ipynb [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+    "print(len(segments))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Choose an Embedding\n",
+    "\n",
+    "Here we're using the GPT2All package and loading one of its smaller models.  Don't worry if you see messages about CUDA libraries being unavailable."
```
Comment (vishakha041) [inline notebooks/semantic_search/website_ingest.ipynb]:
```suggestion
    "Here we're using the GPT4All package and loading one of its smaller models.  Don't worry if you see messages about CUDA libraries being unavailable."
```

**Review by vishakha041 — APPROVED** [review]
Let's update diagram and merge

File: notebooks/semantic_search/cohere_wikipedia_search.ipynb (hunk 2 above)
Comment (bovlb) [reply notebooks/semantic_search/cohere_wikipedia_search.ipynb]: Done.

**Pull request merged**